app/src/main/res/drawable/ic_back.xml

```diff
@@ -0,0 +1,13 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0"
+    android:tint="?attr/colorControlNormal">
+
+    <path
+        android:fillColor="@android:color/white"
+        android:pathData="M11.67,3.87L9.9,2.1 0,12l9.9,9.9 1.77,-1.77L3.54,12z"/>
```

Comment:
Maybe we can save this on this strings.xml file

Reply:
It's ok to have hardcode data and dimens in the xml file of drawables. It defines the resource by it self